2022-06-08 06:09:11 
 HTTPSConnectionPool(host='bearusbank.by', port=443): Max retries exceeded with url: /api/kursExchange?city=%D0%91%D1%80%D0%B5%D1%81%D1%82 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000026A6C95EB80>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))2022-06-08 06:16:35 
 HTTPSConnectionPool(host='bearusbank.by', port=443): Max retries exceeded with url: /api/kursExchange?city=%D0%91%D1%80%D0%B5%D1%81%D1%82 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000024FA767CB50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))2022-06-08 06:18:28 
 HTTPSConnectionPool(host='bearusbank.by', port=443): Max retries exceeded with url: /api/kursExchange?city=%D0%91%D1%80%D0%B5%D1%81%D1%82 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000012191E8DB50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))